docs/pdk_reference.md

```diff
@@ -0,0 +1,83 @@
+## PDK reference
+
+### `pdk new module` command
+
+Generates a new module.
+
+Usage:
+
+```
+pdk new module [--template-url=git_url] [--license=spdx_identifier] module_name [target_dir]
+```
+
+The `pdk new module` command accepts the following arguments and options. Arguments are optional unless otherwise specified.
+
+Argument   | Description   | Values      | Default
+----------------|:---------------:|:------------------:|-------------------------
+`--template-url=git_url` | Overrides the template to use for this module. | A valid Git URL.    | No default.
+`--license=spdx_identifier` | Specifies the license this module is written under. | See https://spdx.org/licenses/ for a list of open source licenses, or use `proprietary`.    | Apache 2.0
```

Comment:
Good point, that `-` is actually rather important when it comes to linting and parsing on the Forge right?